### ISSUE TITLE
Get Pointer to CUDA Stream from Emgu.CV.Cuda.Stream

### ISSUE THREAD

**Issue by Aidan63** [emgucv/emgucv#876]
Hello,

The `Ptr` property of `Emgu.CV.Cuda.Stream` returns a pointer to the `cv::cuda::Stream` object but I want access to the underlying ` cudaStream_t` so I can pass it to other external cuda libraries.

To solve this I created a small helper function to get the cuda stream using opencv's `cudaPtr` function.

```c++
extern "C" __declspec(dllexport) cudaStream_t getCudaStream(cv::cuda::Stream* stream)
{
    return reinterpret_cast<cudaStream_t>(stream->cudaPtr());
}
```

I can then P/Invoke this in C# and have successfully been able to use NPP and Nvdec with a emgu stream.

Would exposing something like this to the emgu stream out the box be useful? I can't be the only one who's needed to do this and having to setup and package a new C++ project just for small functions like this is a bit of a pain.

Thanks.

**Comment by emgucv** [comment]
Fixed in this commit b26be3b

It is included in the 4.8 release.

Thank you for reporting.

**Comment by Aidan63** [comment]
Thanks for the addition! I'll be sure to update and try it out.
